UnityClient/Assets/Scripts/PatternCore/Events/GameEventUI.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using UnityEngine;
+
+public class GameEventUI : SingletonDontDestroy<GameEventUI>
+{
+    public static event Action EventUpdateCurrentMiniMap;
+    public static event Action EventUpdateCoordinateMiniMap;
```

Comment:
The `Event` probably can be omitted from the variable name

Reply:
Event omitted